layouts/partials/footer.html

```diff
@@ -12,28 +12,32 @@
       <div class="col-6 col-sm-4 text-right text-xs-center order-sm-3">
         {{ with $links }}
         {{ with index . "developer"}}
-        {{ template "footer-links-block"  . }}
-        {{ end }}
+        <ul class="list-inline mb-0">
+          {{ template "footer-links-block"  . }}
+            <li class="list-inline-item mx-2" >&nbsp</li>
+            <li class="list-inline-item mx-2" >
+              <img src="/img/powered-gvisor.png" alt="Powered by gVisor">
```

Comment:
Making this part of the footer links unordered list seems kind of strange. What do you think about putting it in its own container below the footer links?

Reply:
This is how I managed to make the logo be on the right side. Either below or above looks strange.

Reply:
Ah, yes this yields the same end result. I thought you meant to to move the image outside the outer <ul>. Done.